### PR TITLE
Created a get-user API and a supertest for the new endpoint

### PR DESCRIPTION
**Pull request by angkimbrell1**
#32

### CODE REVIEW

File: gateway-api/users-api/api/get-user.spec.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const app = require('../app.js');
+const should = require('should');
+const supertest = require('supertest');
+
+describe('/get-user', () => {
+    it('should return a users individual data', done => {
+      supertest(app)
+      .get('/get-user')
+      .end((err, res) => {
+        res.text.should.equal(
```
Comment (stephenthedev) [inline gateway-api/users-api/api/get-user.spec.js]: This is a syntax error. To run your tests, run `npm test` from within the users-api folder.

Also, validations should look like this:

```javascript
res.body.username.should.equal('bob');
```

**Review by stephenthedev — CHANGES_REQUESTED** [review]

**Comment by stephenthedev** [comment]
@angkimbrell1 You are also missing the route in app.js

File: gateway-api/users-api/api/get-user.spec.js (hunk 1 above)
Comment (angkimbrell1) [reply gateway-api/users-api/api/get-user.spec.js]: I've tried this and it gives me a TypeError stating that it cannot read property 'should' of undefined.

File: gateway-api/users-api/api/get-user.spec.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+const app = require('../app.js');
+const should = require('should');
+const supertest = require('supertest');
+
+describe('/get-user', () => {
```
Comment (stephenthedev) [inline gateway-api/users-api/api/get-user.spec.js]: change to /users/:id

File: gateway-api/users-api/app.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+// Allow cross origin requests
+app.use(cors());
+
+// Allow JSON request bodies
+app.use(bodyParser.json());
+
+// Load in all of the routes:
+app.get('/alive', require('./api/alive.js')());
+app.get('/users/:id', require('./api/get-user.js')());
```
Comment (stephenthedev) [inline gateway-api/users-api/app.js]: This file should not be here. Your route should be in the users-api folder, not a new file in the gateway-api

**Review by stephenthedev — CHANGES_REQUESTED** [review]

**Comment by stephenthedev** [comment]
@angkimbrell1 up

File: gateway-api/users-api/app.js (hunk 3 above)
Comment (angkimbrell1) [reply gateway-api/users-api/app.js]: So the users-api folder should not be within the gateway-api folder?

**Pull request merged**